particles/Tutorial/Javascript/README.md

```diff
@@ -51,3 +53,44 @@ defineParticle(({DomParticle, html}) => {
 Once you have Arcs on your computer, run `npm start` in command line, then navigate to [http://localhost:8786/shells/dev-shell/?m=https://$particles/HelloWorld/HelloWorld.arcs](http://localhost:8786/shells/dev-shell/?m=https://$particles/HelloWorld/HelloWorld.arcs)
 
 And that’s it! Congratulations, you have written your first program in Arcs!
+
+# Template Interpolation Exploration
+
+Alright, you’ve made your first Arcs program. But now it’s time to go make something! We are going to need a few more Arcs features to create anything of substance.
+
+To get started, we want a way to make our UI dynamic. Simply returning a static set of HTML is fairly boring. As with our Hello World program, we use `get template()` to return our base set of HTML, however we want to be able to update this in some way. To do so, we use the `render()` method and template interpolation. To make certain we are all speaking the same language, let’s define template interpolation within Arcs:
```

Comment:
"set of HTML" or just "HTML" ? (more than once)

L in HTML stands for language, which is why "set of HTML" reads weird to me.